content/doc/book/managing/plugins.adoc

```diff
@@ -85,6 +85,9 @@ configured Update Center.
 [[install-with-cli]]
 === Using the Jenkins CLI
 
+.Installing Jenkins Plugins using Jenkins CLI
```

Comment:
Pretty sure plugins is lowercase, since it isn't referring to a specific plugin that is a proper noun, such as how it may be listed in the catalog.
```suggestion
.Installing Jenkins plugins using Jenkins CLI
```
```suggestion
.Installing Jenkins Plugins using Jenkins CLI
```

Reply:
Agreed.  Would you like to submit a pull request with that change?